### DISCUSSION TITLE
Add a blog page and example blog posts

### DISCUSSION THREAD

**Discussion by YannHulot** [Etheonor/supanextail#176] [Feedback]
Every sass needs a blog for SEO purposes. I think adding a blog page with SEO tags and proper sitemaps would make this template more attractive. You could also add a few different example blog posts.

**Comment by Etheonor** [comment]
I'm just commenting this one but I agree with all your ideas ^^ I will put that in top of the backlog. Actually the blog was already a branch but I didn't have the type to finish it, definitely something useful. 

I still don't know if I integrate a specific CMS or just markdown/mdx support

**Reply by YannHulot** [reply]
MDX is enough to start with. Then maybe add support later for some CMS but that can come later IMHO.

**Comment by Etheonor** [comment]
Just finished the cleaning, and the blog should be here soon ^^ Right now it's just markdown but I'll see if I can easily add Mdx support

Edit: just added the MDX support

![image](https://user-images.githubusercontent.com/35852770/149964488-675eb0a5-6036-4d34-8cdc-d0773ed04ac0.png)
![image](https://user-images.githubusercontent.com/35852770/149964536-506c11c1-09bb-4ce4-bb42-b69391c34682.png)
